### ISSUE TITLE
AttributeError WLED_WiFi.py

### ISSUE THREAD

**Issue by Geraldidi** [Lord-FEAR/Prismatik-WLED-WiFi#8]
I'm using Windows 10 Python 3.7.0. Prismatik.unofficial.64bit.Setup.5.11.2.23 followed readme and installed modules future, boltons and py-lightpack saved correct values for ip and port in WLED_WiFi.ini

In Prismatik application the plugin list is showing WLED_WiFi (not running)

The result after trying to execute py script is the following:

C:\Program Files\Prismatik\plugins\WLED_WiFi>python WLED_WiFi.py
Traceback (most recent call last):
  File "WLED_WiFi.py", line 38, in <module>
    warls.run()
  File "WLED_WiFi.py", line 26, in run
    d = self.lp.getColoursFromAll()
  File "C:\python37\lib\site-packages\lightpack.py", line 243, in getColoursFromAll
    commands = self._sendAndReceivePayload('getcolors').rstrip(';')\
AttributeError: 'NoneType' object has no attribute 'rstrip'

Any hint on how to make it run? thank you very much

**Comment by Lord-FEAR** [comment]
Try to install the prismatik specified in the description. Perhaps the new version just changed the api

**Comment by Geraldidi** [comment]
This problem was solved. To avoid this problem when following Prismatik wizard I chose Virtual LED device which was a mistake. The correct option was DNRGB (UDP, 1500 LEDs) filled with WLED Controller IP, Port (default 21324) 255 for timeout.  got solution from https://www.youtube.com/watch?v=QJf9ORiWqS8

It worked perfect!  I hope this helps to someone else. I think the process of installing Prismatik should be described in readme

Thanks :)
